### PR TITLE
Adding compute.json for 2019-07-01 api-version with examples carried over from 2019-03-01. Includes newly introduced VirtualMachines.POST.Reapply. 

### PR DESCRIPTION
**Pull request by avinashakka**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/examples/ReimageVirtualMachine copy.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (nschonni) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/examples/ReimageVirtualMachine copy.json]: Looks like this file got copied twice

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/examples/CreateAVmWithPremiumStorage copy.json [hunk 2]
```diff
@@ -0,0 +1,153 @@
+{
```
Comment (nschonni) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/examples/CreateAVmWithPremiumStorage copy.json]: Looks like this file got copied twice

**Comment by erich-wang** [comment]
@avinashakka, to speed up review, please copy files from previous version and submit as first commit, then make necessary change in subsequent commits.

**Review by erich-wang — CHANGES_REQUESTED** [review]
@avinashakka, to speed up review, please copy files from previous version and submit as first commit, then make necessary change in subsequent commits.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/examples/ReimageVirtualMachine copy.json (hunk 1 above)
Comment (avinashakka) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/examples/ReimageVirtualMachine copy.json]: Removed.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/examples/CreateAVmWithPremiumStorage copy.json (hunk 2 above)
Comment (avinashakka) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/examples/CreateAVmWithPremiumStorage copy.json]: Removed.

**Comment by hyonholee** [comment]
@avinashakka Please end this PR to compute-2019-07 branch

**Comment by hyonholee** [comment]
@avinashakka I changed this PR to compute-2019-07 branch.  Please resolve the conflict files.

**Review by hyonholee — CHANGES_REQUESTED** [review]
We need to gather all breaking change for 2019-07-01 version and send a single PR to master.  So I changed this PR to target compute-2019-07 branch.  Then there are conflict files.  Please resolve the conflict files.

**Pull request closed without merging**